spec/defines/interface_spec.rb

```diff
@@ -40,6 +40,40 @@
         it { is_expected.to contain_file("/etc/systemd/network/#{title}.network").without_content(%r{MTUBytes}) }
         it { is_expected.not_to contain_ferm__rule("allow_wg_#{title}") }
       end
+      context 'with required params (peers) and without firewall rules' do
+        let :params do
+          {
+            peers: [
+              {
+                public_key: 'blabla==',
+                endpoint: 'wireguard.example.com:1234',
+              },
+              {
+                public_key: 'foo==',
+                allowed_ips: ['192.0.2.3'],
+              }
+            ],
+            manage_firewall: false,
+            # we need to set destination_addresses to overwrite the default
+            # that would configure IPv4+IPv6, but GHA doesn't provide IPv6 for us
+            destination_addresses: [facts[:networking]['ip'],],
+          }
+        end
+
+        let(:expected_content) do
```

Comment:
Thoughts about renaming this to `expected_netdev_content` and adding a similar test for `expect_network_content` as well to test the other template?

Reply:
i did not changed the network file and it has only a few lines. for me ist current check okay